File: Go/sorting_algorithms/selection_sort.go
package sorting_algorithms

func SelectionSort(list []int) []int {
	for i := 0; i < len(list); i++ {
		sel := list[i]
		selId := i
		for j := i; j < len(list); j++ {
			if list[j] < sel {
				sel = list[j]
				selId = j
			}
		}
		list[i], list[selId] = list[selId], list[i]
	}
	return list
}

File: JavaScript/sorting_algorithms/SelectionSort.js
export default function (list) {
    for (let i = 0; i < list.length; i++) {
        let sel = list[i]
        let selId = i
        for (let j = i; j < list.length; j++) {
            if (list[j] < sel) {
                sel = list[j]
                selId = j
            }
        }
        list[selId] = list[i]
        list[i] = sel
    }
    return list
}